Sources/NIOHTTP2/HTTP2Frame.swift

```diff
@@ -27,9 +27,15 @@ public struct HTTP2Frame: Sendable {
 
     /// Stream priority data, used in PRIORITY frames and optionally in HEADERS frames.
     public struct StreamPriorityData: Equatable, Hashable, Sendable {
-        public var exclusive: Bool
         public var dependency: HTTP2StreamID
+        public var exclusive: Bool
         public var weight: UInt8
+
+        internal init(exclusive: Bool, dependency: HTTP2StreamID, weight: UInt8) {
+            self.exclusive = exclusive
+            self.dependency = dependency
+            self.weight = weight
+        }
```

Comment:
Was the `init` intentionally `internal` before? 

It's a parameter in `Headers.init` but is defaulted to `nil` so it looks like it not being public was an oversight?

Reply:
Seems like an oversight to me as well